apps/web/app/pages/sbtc-rewards/sbtc-rewards.tsx

```diff
@@ -135,8 +138,10 @@ export function SbtcRewards() {
         <Page.Header title="sBTC Rewards" />
 
         <Page.Heading
-          title="Earn yield with Bitcoin on Stacks"
-          subtitle="Get BTC in the form of sBTC on Stacks—the leading L2 for Bitcoin—to earn yield from a variety of reward protocols without sacrificing Bitcoin’s underlying security and self-sovereignty."
+          title={formatPostPrompt(content.posts[postSlug]?.Prompt || '')}
```

Comment:
`Page` should just accept a `postSlug` and read `content` internally, rather than both taking a slug and all of the extra props.